### PR TITLE
Update GitHub repository path

### PR DESCRIPTION
**Pull request by ChocolateLoverRaj**
- Changed `dirname-for-esm` to `dirname-filename-esm`
- Version `1.0.1` to `1.0.2` because of new version with updated GitHub links.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dirname-filename-esm",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (rhysd) [inline package.json]: Please revert this change. Version should be changed by `npm version` by maintainer.

**Review by rhysd — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ChocolateLoverRaj) [reply package.json]: Ok, I reverted the change.

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
Thanks for the updates.
